CHANGELOG.md

```diff
@@ -5,8 +5,15 @@ CHANGELOG
 
 (Capture your changes here)
 
+#### New
+
+#### Bug Fixes
+* Resolved issue where about topic second header was case sensitive. Now insensitive. Issue #174
```

Comment:
GitHub doesn't auto-link issues by numbers in markdown files, only in comments.
Add `[Issue #174]((https://github.com/PowerShell/platyPS/issues/174)`